src/components/HomeCard/HomeCard.js

```diff
@@ -4,28 +4,27 @@ import { Row } from 'react-bootstrap';
 
 import "./HomeCard.css";
 
-const HomeCard = (props) => {
-  const { title, description } = props;
-
-  return (
-    <Row className="my-2 shadow mx-1">
-      <div className="card-title1 w-100 m-0 bg-red text-white pt-2 pl-4 p-1">
-        {title}
-      </div>
-      <div className="bg-light1 w-100">
-        <p className="card-text1 p-4">
-          {description}
-          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nostrum quisquam alias quam?
-        </p>
-      </div>
-    </Row>
-  );
-};
+const HomeCard = ({
+  title,
+  description = '',
```

Comment:
saw you were using `descriptin=""` a lot for this component so I set it to default to n empty string; destructured props from the arg